core/src/components/checkbox/checkbox.ios.vars.scss

```diff
@@ -15,7 +15,7 @@ $checkbox-ios-background-color-on:        ion-color(primary, base) !default;
 $checkbox-ios-background-color-focused:   ion-color(primary, tint) !default;
 
 /// @prop - Size of the checkbox icon
-$checkbox-ios-icon-size:                  dynamic-font-max(26px, 2.38) !default;
+$checkbox-ios-icon-size:                  dynamic-font-max(26px, 2.535) !default;
```

Comment:
Why did this change?

Reply:
I was able to get PixelSnap installed and found that the checkboxes are actually bigger. The max is `66px`.

Reply:
If the max is `66px`, shouldn't this be `2.538`?

Reply:
Done